dev-docs/source/DebuggingUnitTests.rst

```diff
@@ -0,0 +1,94 @@
+Debugging Unit Tests
```

Comment:
Should we have a local table of contents and update some of the section titles to be clearer as to what they are referring to?